### PR TITLE
[KEYCLOAK-17399] - Declarative User Profile

### PR DESCRIPTION
**Pull request by pedroigor**

### CODE REVIEW

File: server_admin/topics/users/user-profile.adoc [hunk 1]
```diff
@@ -0,0 +1,306 @@
+
+
+=== User Profile
+
+In {project_name} a user is associated with a set of attributes. These attributes are used to better describe and identify users within Keycloak as well as to pass over additional information to applications about them.
```
Comment (miquelsi) [inline server_admin/topics/users/user-profile.adoc]: "...within {project_name}..." perhaps?

Comment (pedroigor) [reply server_admin/topics/users/user-profile.adoc]: Fixed and replaced in a few other places too.

File: server_admin/topics/users/user-profile.adoc [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+|Description
+
+|not-empty
+|Check that input value is not empty. It means it is not null for all data types.
+
+|not-blank
+|Check that a string value is not empty. It means it is not null nor empty.
+
+|not-blank
+|Check that a string value is not empty. It means it is not null nor empty.
```
Comment (miquelsi) [inline server_admin/topics/users/user-profile.adoc]: not-blank is duplicated

Comment (pedroigor) [reply server_admin/topics/users/user-profile.adoc]: Done.

**Review by miquelsi — APPROVED** [review]

**Pull request merged**